db/migrations/20190715125635_add_commentable_to_comment.cr

```diff
@@ -0,0 +1,15 @@
+class AddCommentableToComment::V20190715125635 < Avram::Migrator::Migration::V1
+  def migrate
+    alter :comments do
+      add commentable_type : String?
+      add commentable_id : Int64?
+    end
+  end
+
+  def rollback
+    alter :comments do
+      remove :commentable_type
+      remove :commentable_id
```

Comment:
`remove_polymorphic_belongs_to :commentable` would be wonderful to cover this

Reply:
added! 😄